### DISCUSSION TITLE
inside page navigation or hash navigation is not working

### DISCUSSION THREAD

**Discussion by biharihu** [vercel/next.js#17645] [Help]
`
     <Link href="#course">
           <a>
              <p>course</p>
           </a>
    </Link>
`

or 

`
       <a  href="#course">
          <p>course</p>
       </a>
`

tried both of them but not working.. inside page navigation

**Comment by mxmtsk** [comment]
What are you trying to achieve? Please provide a more detailed problem description and a full example.

Normal jump marks work without a problem on normal links within next.js
